### DISCUSSION TITLE
Unnatural Scroll

### DISCUSSION THREAD

**Discussion by benjaminingreens** [thebaselab/codeapp#1017] [Q&A] [answered]
There is an issue (#484) which addresses the unnatural scroll feel in the Monaco editor.

I'm just wondering if there is going to be a fix for this? Scrolling is an integral part of the user experience, and while the app is clearly unfinished in some ways--it is absolutely usable and incredibly useful (could be my new favourite app). The one thing that is stopping me from using it regularly is the scrolling. It just isn't as natural as, say, scrolling on the file list.

**Comment by bummoblizard** [comment, marked as answer]
@benjaminingreens A fix for this is available on TestFlight. Please check the comment I made in #484.
